### DISCUSSION TITLE
Auto resize columns after removing column type

### DISCUSSION THREAD

**Discussion by Drewbty** [dbeaver/dbeaver#34673] [Q&A] [answered]
I have removed 'Show column type icons' as well as 'show attribute filters' from the results grid as I don't need these and want to be able to view more data from the table going across the screen. 

![image](https://github.com/dbeaver/dbeaver/assets/80837965/540964af-39a4-4949-817b-d430b7497d64)

However, auto resize behaviour seems to ignore that I've removed these 2 features from the column headers. 

![image](https://github.com/dbeaver/dbeaver/assets/80837965/38a8c818-4110-4b82-8dfc-706f60226ae3)

We can see that I can easily reduce these column widths from the new space that has been created, which I've done manually in this second screenshot. However Is there no option to do this automatically? I can't seem to find one and seems a pretty standard thing someone would want to do?

![image](https://github.com/dbeaver/dbeaver/assets/80837965/f2c36fa9-11c3-4b1e-8409-174ba2f15b53)

note : fit column width to values won't help me here, I'm trying to fit to (the new) column header width

**Comment by TB-HunBoka** [comment, marked as answer]
Follow this issue for update on this topic #34348
